src/start/PasswordAuthView.js

```diff
@@ -109,6 +112,14 @@ class PasswordAuthView extends React.Component {
           onPress={this.validateForm}
         />
         <ErrorMsg error={error} />
+        <Touchable>
```

Comment:
Do we want whole area to be touchable? We want send `onPress` prop to `Touchable`, right?